subsys/bluetooth/controller/hci_driver.c

```diff
@@ -36,22 +36,14 @@
 LOG_MODULE_REGISTER(bt_sdc_hci_driver);
 
 #if defined(CONFIG_BT_CONN)
-/* It should not be possible to set CONFIG_BT_CTLR_SDC_PERIPHERAL_COUNT larger than
- * CONFIG_BT_MAX_CONN. Kconfig should make sure of that, this assert is to
- * verify that assumption.
- */
-BUILD_ASSERT(CONFIG_BT_CTLR_SDC_PERIPHERAL_COUNT <= CONFIG_BT_MAX_CONN);
-
-#define SDC_CENTRAL_COUNT (CONFIG_BT_MAX_CONN - CONFIG_BT_CTLR_SDC_PERIPHERAL_COUNT)
 
-#else
-
-#define SDC_CENTRAL_COUNT 0
+BUILD_ASSERT(CONFIG_BT_CTLR_SDC_PERIPHERAL_COUNT <= CONFIG_BT_MAX_CONN);
+BUILD_ASSERT(CONFIG_BT_CTLR_SDC_CENTRAL_COUNT <= CONFIG_BT_MAX_CONN);
```

Comment:
I guess we do not need those build asserts now as it is already validated in KConfig?